packages/web-components/src/components/ic-top-navigation/ic-top-navigation.tsx

```diff
@@ -327,15 +332,25 @@ export class TopNavigation {
                           <slot name="app-icon" />
                         </div>
                       )}
-                      <ic-typography variant={appTitleVariant}>
-                        <h1 class="title-wrap">
-                          {isSlotUsed(this.el, "app-title") ? (
-                            <slot name="app-title"></slot>
-                          ) : (
-                            this.appTitle
-                          )}
-                        </h1>
-                      </ic-typography>
+                      {this.deviceSize <= DEVICE_SIZES.S &&
+                      this.shortTitle !== "" ? (
+                        <ic-typography
+                          variant="subtitle-small"
+                          aria-label={`${this.appTitle} (${this.shortTitle})`}
+                        >
+                          {this.shortTitle}
```

Comment:
Also, this is not wrapped in a heading tag whereas when it is on a bigger device, it is.